examples/heat_square.ipynb

```diff
@@ -139,7 +139,7 @@
     }
    ],
    "source": [
-    "K = create_sparsity_pattern(dh);\n",
```

Comment:
I kinda like `create` :P Otherwise it feels like you are just using a getter for the `sparsity_pattern` that exists in `dh`?